lib/src/configuration.dart

```diff
@@ -85,6 +85,8 @@ class TestConfiguration {
     this.createWorld,
   });
 
+  int stepMaxRetries = 3;
```

Comment:
We should add documentation to this parameter.  Just something short.  I actually think this should default to zero to preserve existing behaviour.  It is actually bad practice to retry.  With cucumberjs to have to be explicit about the number of retries for a step / scenario